lib/extended-css-selector.js

```diff
@@ -96,18 +96,17 @@ var ExtendedSelector = (function () { // jshint ignore:line
      * Joins tokens values
      */
     var joinTokens = function(selector, relationToken, tokens) {
```

Comment:
You might want to point out in the comment that you use string concatenation due to browsers optimizations